Camera/ListViewViewController.swift

```diff
@@ -27,8 +25,9 @@ class ListViewViewController: UIViewController, UITableViewDataSource, UITableVi
     }
     
     func updateTableView() {
-        clips = returnContentsOfTemporaryDocumentsDirectory()
-        clipCount = clips.count
+        let realm = try! Realm()
+        clips = realm.objects(Clip).sorted("filename", ascending: false)
```

Comment:
I like that this is explicit now. And was thinking we could perhaps have the most recent clips at the top of the list but export and preview them all together in chronological order.